### PR TITLE
Renames wirecutters 'wire stripper'

### PR DESCRIPTION
**Pull request by pudl**
:cl:
rscadd: Wirecutters are now named wire stripper.
/:cl:

### CODE REVIEW

**Comment by Cheridan** [comment]
good change, from the sprite its obvious that they're more like these:

http://img.directindustry.com/images_di/photo-g/22232-5883333.jpg

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -133,8 +133,8 @@
  * Wirecutters
  */
 /obj/item/weapon/wirecutters
-	name = "wirecutters"
-	desc = "This cuts wires."
+	name = "wire strippers"
```
Comment (optimumtact) [inline code/game/objects/items/weapons/tools.dm]: Wire Strippers surely

Comment (Cheridan) [inline code/game/objects/items/weapons/tools.dm]: we dont capitalize items

Comment (optimumtact) [inline code/game/objects/items/weapons/tools.dm]: :cry: :no_good: :sob: 

:credit_card: :+1:

Comment (pudl) [inline code/game/objects/items/weapons/tools.dm]: DREAM KILLER

**Comment by pudl** [comment]
add a do not merge tag please

**Comment by Cheridan** [comment]
what, why. you got more planned?

**Comment by PKPenguin321** [comment]
:-1: Reeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee they've been wirecutters since literally forever

**Comment by Supermichael777** [comment]
change the sprite not the name

**Comment by Dorsisdwarf** [comment]
I really don't get this PR

**Comment by MMMiracles** [comment]
I mean wire cutters have an insulated grip to prevent shock. In-game wire cutters don't insulate at all.

**Comment by Firecage** [comment]
But...we don't use them to strip wires?

**Comment by xxalpha** [comment]
Instead of fixing bugs, people are just removing things and renaming other things that have been since forever.

**Comment by Cheridan** [comment]
>they don't want strippers on their space station

wow :dancer:

**Comment by ike709** [comment]
feature freeze is not an excuse to alter muh tg culture

**Comment by pudl** [comment]
please remove the do not merge tag

**Comment by Iamgoofball** [comment]
first hoes
now strippers

what next in /degeneracy/station

**Comment by Xhuis** [comment]
This is why nobody likes feature freezes, people

**Comment by PKPenguin321** [comment]
I like freezes when we actually fix bugs this freeze has been cancer though

**Comment by pudl** [comment]
**WHAT ARE YOU DOING IN MY PR**

**Comment by PKPenguin321** [comment]
What is your pr doing in my freeze

**Comment by pudl** [comment]
its not YOUR freeze its OUR freeze

**Comment by Cheridan** [comment]
sorry it appears that this is 

U N I V E R S A L L Y  D I S L I K E D

**Comment by pudl** [comment]
/tg/ isnt ready for redpilled tool names yet

**Comment by Iamgoofball** [comment]
dont pr to me or my repo ever again

**Comment by Bawhoppen** [comment]
BTW, Many wirecutters have dual utility, and can do both.

**Comment by Xhuis** [comment]
necro of the millenium

**Pull request closed without merging**